gradle/deps.properties

```diff
@@ -1,3 +1,3 @@
 version.project=0.0.1-SNAPSHOT
-version.aya=0.19-SNAPSHOT
+version.aya=0.19
```

Comment:
Not 0.20?

Reply:
It needs guest0x0 to disable gradle metadata